test/transforms.jl

```diff
@@ -55,10 +61,45 @@
     tₒ = revert(T, n, c)
     @test t == tₒ
 
+    # selection with tuple of strings
+    T = Select(("d", "c", "b"))
+    n, c = apply(T, t)
+    @test Tables.columnnames(n) == (:d, :c, :b)
+    tₒ = revert(T, n, c)
+    @test t == tₒ
+
+    # selection with vector strings
+    T = Select(["d", "c", "b"])
+    n, c = apply(T, t)
+    @test Tables.columnnames(n) == (:d, :c, :b)
+    tₒ = revert(T, n, c)
+    @test t == tₒ
+
+    # selection with Regex
+    T = Select(r"[dcb]")
+    n, c = apply(T, t)
+    @test Tables.columnnames(n) == (:b, :c, :d) # a ordem das colunas é a mesma da table original - obs: traduzir
```

Comment:
Please use English always, even in comments.

Reply:
Done.